package.json

```diff
@@ -31,7 +31,6 @@
   "homepage": "https://github.com/ripple/validator-history-service#readme",
   "devDependencies": {
     "@types/axios": "^0.14.0",
-    "@types/create-hash": "^1.2.2",
```

Comment:
why didnt package-lock change with this?

Reply:
Turns out create-hash is still used by `xrpl-validator-domains`.  That library needs love.  It is using xrpl@3 still.